### ISSUE TITLE
download model

### ISSUE THREAD

**Issue by bberry17** [yulunzhang/RCAN#64]
Hi, I tried downloading all the models from the below Dropbox link, but it didn't work.
It said, 'there is no file exist.'
"All the models (BIX2/3/4/8, BDX3) can be downloaded from Dropbox and BaiduYun."

Where can I get the whole model?

**Comment by yulunzhang** [comment]
Hi,
I just check the links I provided in the repo and they are available. Here're the links
https://www.dropbox.com/s/qm9vc0p0w9i4s0n/models_ECCV2018RCAN.zip?dl=0
https://pan.baidu.com/s/1bkoJKmdOcvLhOFXHVkFlKA

**Comment by bberry17** [comment]
Hi, thanks for the reply. 
I tried the new link you added, but it doesn't work, too. 
I downloaded a zip.file directly from the dropbox and saved it in my dropbox, but those didn't work.
The message was pop-up with "There is an error to download file/."
Also, I tried the new link from Baidu, and it shows the message as "A new version of the client has been installed and still cannot be downloaded for reasons."

I don't get what problem in the links...
